### DISCUSSION TITLE
More thumbnail size options

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#35] [Ideas]
Maybe adding more thumbnail sizes like small,medium,big for fitting more user needs would be nice
Current Thumbnail size is OK I guess 
and this is not a big deal at all . 
I'm just quoting ideas here

**Comment by Radiokot** [comment]
I'm thinking of adding the preference to change the minimal thumbnail size. Currently, it is 2 cm.

**Comment by ChrisKros** [comment]
Maybe it would be also convenient to change the thumbnail size with pinch to zoom gestures.

**Reply by Radiokot** [reply]
Sure

**Comment by Radiokot** [comment]
The ability to change the item size is added to the preferences in [1.24.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.24.0(37)).
Pinch gesture will be added later.
